src/styled/layouts/admin-layout.ts

```diff
@@ -67,4 +69,15 @@ export const IconActionButton = styled(Button, {
 })(({ theme }) => ({
   borderRadius: 25,
   padding: theme.spacing(1.5)
+}));
+
+/**
+ * Styled new button component
+ */
+export const NewButton = styled(Button, {
+  label: "new-button"
+})(() => ({
```

Comment:
Name should be more describing, for example "NewUserButton".

Reply:
Changed it to NewUserButton